plugins/publish-plugin/src/main/kotlin/MultiplatformAppyxPublishPlugin.kt

```diff
@@ -7,20 +6,33 @@ import org.gradle.api.publish.maven.MavenPublication
 import org.gradle.api.tasks.bundling.Jar
 
 internal class MultiplatformAppyxPublishPlugin : ProjectPlugin() {
+    override fun apply(project: Project) {
+        super.apply(project)
+        project.gradle.projectsEvaluated {
+            if (project.name == "common") {
+                throw IllegalArgumentException(
+                    "Multiplatform project with path ${project.projectDir} should not have its name as common." +
+                            " This will make artifacts rewrite each other." +
+                            " Please change the name of the project in settings.gradle"
+                )
+            }
```

Comment:
Nice! 👍 

Maybe any advice in the exception message what naming pattern to apply (or where to find the pattern)?

Reply:
Naming is unique per project, probably throwing an error when having common as a name should suffice